slobrok/src/vespa/slobrok/server/proxy_map_source.h

```diff
@@ -0,0 +1,26 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "map_source.h"
+#include <set>
+
+namespace slobrok {
+
+class ProxyMapSource : public MapSource, public MapListener {
```

Comment:
why do we need this? (why not just let the leaf map listeners listen directly to the original map source?)